upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -729,6 +731,11 @@ func setupZones(opt *NewClusterOptions, cluster *api.Cluster, allZones sets.Stri
 				return nil, err
 			}
 		}
+
+	case api.CloudProviderScaleway:
+		if len(opt.Zones) > 1 {
+			return nil, fmt.Errorf("scaleway cloud provider currently supports only one zone (location)")
+		}
```

Comment:
Not sure the "location" terminology is correct here or if you want clusters to only allow 1 AZ (zone).
Should this work for Scaleway?
```
kops create cluster --zones=fr-par-1,fr-par-2,fr-par-3
```